### PR TITLE
Feature/loadbalancing fix get

### PR DESCRIPTION
**Pull request by evhenyshaliov**
Updated get methods based on investigation of Pavel Mitrafanau for serverless mode. Get methods should work with fields from Stackdriver logs

### CODE REVIEW

**Comment by aleonidex** [comment]
Hi @jondavidi please have a look at this small fix, the last remainder of Load Balancer epic.

File: tools/c7n_gcp/c7n_gcp/resources/loadbalancer.py [hunk 1]
```diff
@@ -342,4 +356,28 @@ class resource_type(TypeInfo):
...
+        @staticmethod
+        def get(client, resource_info):
+            return client.execute_command('get', {
+                'project': resource_info['project_id'],
+                'region': resource_info['location'],
+                'backendService': resource_info[
+                    'resourceName'].rsplit('/', 1)[-1]})
+
+    def get_resource_query(self):
+        if 'query' in self.data and self.data.get('query')[0].__contains__('region'):
```
[the remark this answers is not in the document]
Comment (evhenyshaliov) [reply tools/c7n_gcp/c7n_gcp/resources/loadbalancer.py]: This part was deleted

File: tools/c7n_gcp/tests/test_loadbalancer.py [hunk 2]
```diff
@@ -516,15 +594,55 @@ def test_loadbalancer_global_address_query(self):
...
+            session_factory=factory)
+        exec_mode = p.get_execution_mode()
+        event = event_data('lb-global-addresses-get.json')
+        instances = exec_mode.run(event, None)
+        self.assertEqual(len(instances), 1)
+        self.assertEqual(instances[0]['kind'], 'compute#address')
+        self.assertEqual(instances[0]['name'], 'custodian-global-address-0')
+
+
+class LoadRegionBalancingBackendServiceTest(BaseTest):
```
Comment (jondavidi) [inline tools/c7n_gcp/tests/test_loadbalancer.py]: this has a typo. needs to be `LoadBalancingRegionBackendServiceTest(BaseTest):`

File: tools/c7n_gcp/c7n_gcp/resources/loadbalancer.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jondavidi) [reply tools/c7n_gcp/c7n_gcp/resources/loadbalancer.py]: I'm still seeing it in the diff? maybe it's loading wrong version for me but I still see the same line. why was it supposed to be removed and what replaced it?

File: tools/c7n_gcp/tests/test_loadbalancer.py (hunk 2 above)
Comment (evhenyshaliov) [reply tools/c7n_gcp/tests/test_loadbalancer.py]: It is deleted. Please check

File: tools/c7n_gcp/tests/data/events/lb-region-backend-services-patch.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
+{
```
Comment (jondavidi) [inline tools/c7n_gcp/tests/data/events/lb-region-backend-services-patch.json]: can't this file be deleted now? I think this resource was deleted from the PR. if so, then please remove this along with any other similar recording files that can be deleted from this PR. we don't want to include extra unnecessary recordings

**Comment by evhenyshaliov** [comment]
3 JSON files of regionBackendService were deleted

**Pull request closed without merging**